controllers/apply-next/views/questions-list.njk

```diff
@@ -0,0 +1,110 @@
+        Provide a date
+    {% elseif field.type === 'email' %}
+        Provide an email address
+    {% elseif field.type === 'file' %}
+        Upload a file
+    {% elseif field.type === 'number' %}
+        Provide a number
+    {% elseif field.type === 'tel' %}
+        Provide a telephone number
+    {% elseif field.type === 'text' %}
+        Provide a sentence or two
+    {% elseif field.type === 'textarea' %}
+        Provide a longer answer
+        {% if field.settings.minWords and field.settings.maxWords %}
+            (between {{ field.settings.minWords }}-{{ field.settings.maxWords }} words)
+        {% else %}
+            {% if field.settings.minWords %} (minimum of {{ field.settings.minWords }} words){% endif %}
+            {% if field.settings.maxWords %} (maximum of {{ field.settings.maxWords }} words){% endif %}
+        {% endif %}
+    {% endif %}
+    {% if step.notRequired %}
+        &mdash; optional, but may be required depending on your circumstances
+    {% elseif field.isRequired %}
+        &mdash; required
+    {% endif %}
+{% endmacro %}
+
+{% macro header() %}
+    <div class="u-tone-background-tint u-padded">
+        {{ caller() }}
+    </div>
+{% endmacro %}
+
+{% macro question(first = false) %}
+    <div class="{% if not first %}u-padded-top u-border-top--thin u-margin-top{% endif %}">
+        {{ caller() }}
+    </div>
+{% endmacro %}
+
+{% macro questionList(form) %}
+
+    {% call header() %}
+        <h3>Eligibility questions</h3>
+        <p class="u-no-margin application-text-minor">You'll need to answer all of these before beginning your application form.</p>
+    {% endcall %}
+    <div class="u-padded">
+        {% for field in form.eligibilityQuestions %}
+            {% call question(first = loop.first) %}
+                <h5>{{ field.question }}</h5>
+                {% if field.explanation %}
+                    <p class="s-prose application-text-minor">{{ field.explanation | safe }}</p>
+                {% endif %}
+            {% endcall %}
+        {% endfor %}
+    </div>
+
+    {% for section in form.sections %}
+        <div class="u-tone-background-tint u-padded">
+            <h3>Section {{ loop.index }}: {{ section.title }}</h3>
+            {% if section.introduction %}
+                <p class="u-no-margin application-text-minor">{{ section.introduction }}</p>
+            {% endif %}
+        </div>
+        <div class="u-padded">
+            {% for step in section.steps %}
+                <h4 class="t--underline{% if not loop.first %}  u-margin-top-l{% endif %}">
+                    Step {{ loop.index }}: {{ step.title }}
+                </h4>
+                {% for fieldset in step.fieldsets %}
+                    {% for field in fieldset.fields %}
+                        <div class="{% if not loop.first %}u-padded-top u-border-top--thin u-margin-top{% endif %}">
+                            <h5>{{ field.label }}</h5>
```

Comment:
Not sure headings are the appropriate markup here, is this a definition list of sorts?

Reply:
If you like – I thought each question would be a subheading in the document outline, but I'll turn it into DLs/DDs.

Reply:
Ah, I think I see. In which case it might just be the help text that feels semantically wrong, it should probably be a plain paragraph rather than a `<h6>`. The help text shouldn't create a new heading-level should it?

Reply:
Went with DLs in the end!